src/studio/src/designer/backend/package.json

```diff
@@ -10,14 +10,16 @@
     "gulp-copy-files": "gulp copy-files",
     "gulp-build-ux-editor": "gulp build-ux-editor",
     "gulp-develop-dashboard": "gulp develop-dashboard",
-    "gulp-develop-designer-frontend": "gulp develop-designer-frontend"
+    "gulp-develop-designer-frontend": "gulp develop-designer-frontend",
+    "prepare": "cd ../../../../../ && husky install"
```

Comment:
Have you tested this in several terminals?

Reply:
I have tested it on Windows Powershell and git bash. This is also the recommended way to install husky when package.json is not on root (https://typicode.github.io/husky/#/?id=custom-directory).

We get a package.json on root as part of this PR because the hooks that are executed need some packages to be installed. I dont  think it makes sense to move husky installation to that, because a common workflow would not include executing `npm install` on root.

Do you see any potential issues with this approach?